srcpkgs/aspell-it/template

```diff
@@ -0,0 +1,14 @@
+# Template file for 'aspell-it'
+pkgname=aspell-it
+version=2.4
+revision=1
+_version="${version}-20070901-0"
```

Comment:
The commit message is not correct anymore.

Reply:
Now it is again.